assets/admin/tour/course-tour/steps.js

```diff
@@ -65,6 +155,36 @@ function getTourSteps() {
 			referenceElements: {
 				desktop: '',
 			},
+			action: async () => {
+				await ensureLessonBlocksIsInEditor();
+				await performStepActionsAsync( [
+					{
+						action: () => {
+							focusOnCourseOutlineBlock();
+							const lesson = document.querySelector(
+								'[data-type="sensei-lms/course-outline-lesson"]'
+							);
+							if ( lesson ) {
+								lesson.focus();
```

Comment:
I imagine this previous focus is here for some edge case, right? Could you also document this one similar to the others you added a comment? I was thinking about this case, but I couldn't identify it. haha

Reply:
Yeah good point. I just added it there to scroll to the lesson area in case it's out of sight. But focusing on lesson does that as well. So I've removed the previous call here https://github.com/Automattic/sensei/pull/7535/commits/70335906470c0751aa76b77c2ec1afefd914b511